modules/home/assets/js/components/create-new-page-dialog.js

```diff
@@ -0,0 +1,75 @@
+import DialogHeader from '@elementor/ui/DialogHeader';
```

Comment:
Could we group some of these?

e.g. 

```suggestion
import { DialogHeader, DialogHeaderGroup ... } from '@elementor/ui';
```